cmd/juju/commands/main.go

```diff
@@ -63,13 +63,51 @@ func init() {
 //  Main
 
 var jujuDoc = `
-Juju provides easy, intelligent application orchestration on top of cloud
-infrastructure providers such as Amazon EC2, MaaS, OpenStack, Windows, Azure,
-or your local machine.
+Juju provides easy, intelligent application orchestration on top of Kubernetes,
+cloud infrastructure providers such as Amazon, Google, Microsoft, Openstack,
+MAAS (and more!), or even on your local machine via LXD.
 
-See https://discourse.jujucharms.com/ for ideas, documentation and FAQ.
+See https://juju.is for getting started tutorials and additional documentation.
 `
 
+var usageHelp = `
+Juju provides easy, intelligent application orchestration on top of Kubernetes,
```

Comment:
I rephrased this block to make it a bit more "catchy" and to hint that _we have getting started tutorials too_. Let me know if we need any changes to the wording.